1.wave2_data/3.processes_features/extract_image_features_interstellar.py

```diff
@@ -83,10 +87,60 @@
 image_features_df.head()
 
 
-# ## View info of the dataframe
+# ## Merge platemap metadata with extracted image features
 
 # In[8]:
 
 
+image_features_df = annotate(
+    profiles=image_features_df,
+    platemap=platemap_df,
+    join_on=["Metadata_well", "Image_Metadata_Well"],
+    output_file="none",
+)
+
+
+# ## Add condition as a metadata column to the dataframe
+
+# In[9]:
+
+
+def conditions(well_column):
+    """
+    function works to add condition based on the well column number for each single cell (row)
+    """
+    if well_column == 5 or 8:
+        return 8
+    if well_column == 6 or 7:
+        return 7
+
```

Comment:
Consider providing an `else` case for scenarios where a value may not be returned based on the if cases.

Reply:
Ooo good point!

Reply:
@d33bs Would adding a print statement work?